src/main/java/org/spongepowered/api/text/translation/locale/Locales.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.text.translation.locale;
+
+import org.spongepowered.api.Sponge;
+
+import java.util.Locale;
+
+/**
+ * The list of locales available in Vanilla Minecraft.
+ */
+public final class Locales {
+
+    private Locales() {
+    }
+
+    /**
+     * Gets a locale for the specified locale code, e.g. {@code en_US}.
+     *
+     * @param locale The locale to lookup (e.g. {@code en_US}.
+     * @return The locale
+     */
+    public static Locale of(String locale) {
+        return Sponge.getRegistry().getLocale(locale);
+    }
+
+    public static final Locale AF_ZA = new Locale("af", "ZA");
+    public static final Locale AR_SA = new Locale("ar", "SA");
+    public static final Locale AST_ES = new Locale("ast", "ES");
+    public static final Locale AZ_AZ = new Locale("az", "AZ");
+    public static final Locale BG_BG = new Locale("bg", "BG");
+    public static final Locale CA_ES = new Locale("ca", "ES");
+    public static final Locale CS_CZ = new Locale("cs", "CZ");
+    public static final Locale CY_GB = new Locale("cy", "GB");
+    public static final Locale DA_DK = new Locale("da", "DK");
+    public static final Locale DE_DE = Locale.GERMANY;
+    public static final Locale EL_GR = new Locale("el", "GR");
+    public static final Locale EN_AU = new Locale("en", "AU");
+    public static final Locale EN_CA = new Locale("en", "CA");
```

Comment:
Isn't there an `FR_CA` as well? so `CANADA` wouldn't really quite 100% make sense in this case.